Sources/Composed/Providers/ComposedSectionProvider.swift

```diff
@@ -100,6 +100,32 @@ open class ComposedSectionProvider: AggregateSectionProvider, SectionProviderUpd
         return -1
     }
 
+    public func sectionOffset(for section: Section) -> Int {
```

Comment:
Why am I only seeing additions here? I was expecting to see 'changes' or other deletions?

Reply:
This is a new function, previously there was only a `sectionOffset(for:)` for `SectionProvider`s

Reply:
ahhh... I missed that! To be honest, I don't think I changed this since you originally implemented it, so definitely agree on more surface area test coverage.

Reply:
Building up some tests for the existing code would be good and enforcing decent tests for new code should hopefully stop mistakes like this from slipping through again